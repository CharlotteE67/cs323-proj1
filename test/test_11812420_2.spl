int func_1(){
  return (a||b);
}
float 1_invalid_func(){
  return a;
}
int test(int n) {
  int a = _test234;
  int test_func = func_1();

  char invalid_char_1 = '\xu4';
  char invalid_char_2 = '\x1234d';

  float invalid_float_1 = .00234;
  float invalid_float = 00.4268;
}
