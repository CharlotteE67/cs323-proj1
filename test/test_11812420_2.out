Error type A at Line 4: identifier starts with number
Error type A at Line 11: hexadecimal representation with illegal char
Error type A at Line 12: hexadecimal representation too long 
Error type A at Line 14: float number missing digits
Error type A at Line 15: float number start with 0
